concepts/Common/Integration Guides/70 Manage Azure Blob Storage with DevExtreme Components/30 FileManager.md

```diff
@@ -0,0 +1,203 @@
+Full code examples:
+
+* [Client-side data handling](https://github.com/DevExpress-Examples/devextreme-file-manager-azure-client-side-binding)
+* [Server-side data handling](https://github.com/DevExpress-Examples/devextreme-file-manager-azure-server-side-binding)
+
+Azure Blob Storage doesn't expose a traditional file system to the end user. When you request to view the list of blob entries, Azure returns XML: 
+
+    <?xml version="1.0" encoding="utf-8"?>
+    <EnumerationResults ServiceEndpoint="https://aspxdemos.blob.core.windows.net/" ContainerName="testfilemanager">
+        <Blobs>
+            <Blob>
+                <Name>1.jpg</Name>
+                <Properties>
+                    <Creation-Time>Mon, 04 Dec 2023 07:24:58 GMT</Creation-Time>
+                    <Last-Modified>Wed, 06 Dec 2023 06:29:09 GMT</Last-Modified>
+                    <Etag>0x8DBF624A7904382</Etag>
+                    <Content-Length>3126</Content-Length>
+                    <Content-Type>application/octet-stream</Content-Type>
+                    <Content-Encoding />
+                    <Content-Language />
+                    <Content-CRC64 />
+                    <Content-MD5 />
+                    <Cache-Control />
+                    <Content-Disposition />
+                    <BlobType>BlockBlob</BlobType>
+                    <LeaseStatus>unlocked</LeaseStatus>
+                    <LeaseState>available</LeaseState>
+                    <ServerEncrypted>true</ServerEncrypted>
+                </Properties>
+                <OrMetadata />
+            </Blob>
+    ... (other Blob records)
+        </Blobs>
+        <NextMarker />
+    </EnumerationResults>
+
+The `FileManager` component cannot work with such data. We need to perform additional actions to generate a FileManager-compatible filesystem representation.
+
+We can perform this task on the client or on the server.
```

Comment:
We should update this line without using "perform". Do you mean "run this task"?